### PR TITLE
feat(ion-cannon): add Ion Cannon Sound

### PR DESCRIPTION
**Pull request by EikeSan**
Added the sound effect for ion cannon and changed the component factory of it.

### CODE REVIEW

File: src/main/java/org/openRealmOfStars/player/ship/ShipComponentFactory.java [hunk 1]
```diff
@@ -2512,21 +2512,21 @@ private static ShipComponent createWeapon(final int index) {
       tmp.setEnergyRequirement(1);
     }
     if (index == COMPONENT_ION_CANNON_MK2) {
-      tmp = new ShipComponent(index, "Ion cannon Mk2", 5, 3,
```
Comment (tuomount) [inline src/main/java/org/openRealmOfStars/player/ship/ShipComponentFactory.java:2515]: Why did you changed production and metal cost of the ion cannon?

**Review by tuomount — COMMENTED** [review]
You need to add credits for Ion cannon effect and mention correct CC license for that. That is done by updating CreditsView.java. When that is being added you can run the Oros with parameter "--credits" and copy paste output to AUTHORS.md.

Sound effect itself is awesome!

**Comment by EikeSan** [comment]
@tuomount  I added the credits and rollback the costs values

**Comment by tuomount** [comment]
Could you please link to original sound. I couldn't found ion cannon sound made by The Motion Monkey.

**Comment by EikeSan** [comment]
Hello, 
You right, i had an old folder into my hd named the motion monkey with that sound, i think i get from other place but i cant trace it back, I'm so sorry for this :disappointed: and sorry for took so long. I get other sound to replace that one, take a look and give-me a feedback ^^

**Pull request merged**